index.html

```diff
@@ -4,52 +4,62 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Setup Mobile First</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css" />
     <script
       src="https://kit.fontawesome.com/b1fa342dbf.js"
-      crossorigin="anonymous"
-    ></script>
+      crossorigin="anonymous"></script>
   </head>
-
   <body>
     <header>
       <div class="header flex">
         <a href="#" class="brand">Faiza</a>
         <nav>
-        <i class="fa-sharp fa-solid fa-bars"></i>
+          <i class="fa-sharp fa-solid fa-bars icons"></i>
         </nav>
-        <div>
+      </div>
     </header>
     <main>
       <section class="home flex">
         <div class="cont-parent flex">
-        <div class="content flex">
-          <h1>
-            Hey There. <br />
-            I'm Faiza.
-          </h1>
-          <h3>I'm Web Developer</h3>
-          <p>
-            I can help you to build a forms , feature or website. Look through
-            some of my work and experience! If you like my work contect me
-                   don't hesitate.
-          </p>
+          <div class="content flex">
+            <h1>
+              Hey There. <br />
+              I'm Faiza.
+            </h1>
+            <h3>I'm Web Developer</h3>
+            <p>
+              I can help you to build a forms , feature or website. Look through
+              some of my work and experience! If you like my work contect me
+              don't hesitate.
+            </p>
+            <img class="group" src="./img/Group.png" alt="" />
+          </div>
+          <div class="socialmedia flex">
+            <ul>
+              <li class="icons">
+                <a href="#"
+                  ><img src="./img/angellist.svg" alt="angetlist"
+                /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="./img/github.svg" alt="github" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"
+                  ><img src="./img/Linkedin icon.svg" alt="Linkedin"
+                /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="./img/medium.svg" alt="medium" /></a>
+              </li>
+              <li class="icons">
+                <a href="#"><img src="./img/twitter.svg" alt="twitter" /></a>
+              </li>
+            </ul>
```

Comment:
- [ ] Kindly change the color of your social icons to match the given template. I am adding this as a required suggestion because you had it right the first time, and the color only changed after the first review. Kindly make it white again.
### Your design
![image](https://user-images.githubusercontent.com/94551484/235005715-43cc2f5f-3ac1-4fdf-b7e7-a3b4c2727139.png)
### figma
![image](https://user-images.githubusercontent.com/94551484/235005745-7e569efa-f09e-463a-9d45-1a9a83a08cca.png)